package co.nilin.opex.auth.exception

class UserAlreadyExistsException(message: String) : RuntimeException(message) 